Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -281,47 +281,13 @@ + (WSSInMemoryRandomAccessStream*)_populateStream:(WSSInMemoryRandomAccessStream
 + (WSSInMemoryRandomAccessStream*)_grabStreamFromUIImage:(UIImage*)image {
```

Comment:
>_grabStreamFromUIImage [](start = 34, length = 22)

in WocCatalog, the last test is copy and Paste. please run through that to make sure image copy & paste is still working.